ipatests/test_integration/test_commands.py

```diff
@@ -1295,3 +1295,25 @@ def test_ipa_nis_manage_enable_incorrect_password(self):
         )
         assert result.returncode == 1
         assert msg in result.stderr_text
+
+    def test_pkispawn_log_is_present(self):
+        """
+        This testcase checks if pkispawn logged properly.
+        It is a candidate from being moved out of test_commands.
+        """
+        result = self.master.run_command(
+            ["ls", "/var/log/pki/"]
+        )
+        pkispawnlog = None
+        for file in result.stdout_text.splitlines():
+            if file.startswith("pki-ca-spawn"):
+                pkispawnlog = file
+                break
+        assert pkispawnlog is not None
+        result = self.master.run_command(
+            ["cat", "/var/log/pki/{}".format(pkispawnlog)]
+        )
```

Comment:
Now that you have a whole filename you can use get_file_contents()

Reply:
Done thanks